tests/templates/kuttl/backup-restore/21-install-opensearch-1.yaml.j2

```diff
@@ -166,23 +163,6 @@ spec:
           - name: system-trust-store
             emptyDir:
               sizeLimit: 10Mi
-          - name: tls
-            ephemeral:
-              volumeClaimTemplate:
-                metadata:
-                  annotations:
-                    secrets.stackable.tech/scope: node,pod,service=opensearch-1,service=opensearch-1-nodes-default,service=opensearch-1-nodes-default-headless,listener-volume=listener
-                    secrets.stackable.tech/class: tls
-                spec:
-                  storageClassName: secrets.stackable.tech
-                  accessModes:
-                    - ReadWriteOnce
-                  resources:
-                    requests:
-                      storage: "1"
-          - name: tls-concatenated
-            emptyDir:
-              sizeLimit: 1Mi
```

Comment:
The volume `tls-concatenated` is used by the `init-tls` container to add the admin certificate and share the concatenated tls CA with the `opensearch` container.